tests/src/test/scala/whisk/core/cli/test/Swift3Tests.scala

```diff
@@ -72,15 +74,15 @@ class SwiftTests
      *
      * It is ignored because Swift3 is experimental. The test is failed sometimes and breaks the CI pipeline. This was agreed.
      */
-    ignore should "invoke a swift:3 action" in withAssetCleaner(wskprops) {
+    ignore should "invoke a swift action with a network call" in withAssetCleaner(wskprops) {
```

Comment:
This test is redundant.  We're doing the same thing in Swift3ActionContainerTests.  I'll remove it here.